packages/api/test/utils/checkAgainstSpec.ts

```diff
@@ -19,17 +19,40 @@ ajv.addKeyword({
 
 ajv.addFormat("hex", /^0x[a-fA-F0-9]+$/);
 
+export type Operation = {
+  id: string;
+  required?: string[];
+};
+
+/**
+ * Run all tests defined as part of the OpenAPI spec provided as `openApiJson`.
+ *
+ * @param openApiJson
+ * @param routesData
+ * @param reqSerializers
+ * @param returnTypes
+ * @param testDatas
+ * @param opts
```

Comment:
jsdoc like this is not very informative, the only param that is documented here is `filteredOperations`, alternatively could create a type for that and add jsdoc there

Reply:
Makes sense.